server/index.js

```diff
@@ -97,6 +97,11 @@ module.exports = async (_config = {}) => {
   const http = require('http');
   const express = require('express');
 
+  // network information
+  const bcoinNetworks = require('bcash').protocol.networks;
+  const bcashNetworks = require('bcash').protocol.networks;
+  const hsdNetworks = require('hsd').protocol.networks;
+
```

Comment:
Curious what you think of doing something like:

```
const networks = {
  bitcoin: require('bcoin/protocol/networks'),
  bitcoincash: require('bcash/protocol/networks'),
  handshake: require('hsd/protocol/networks'),
}
```

Then there are less global variables and it feels a little cleaner. This pattern could also work well in other plugins when we need to include the package of each protocol

Reply:
If we followed this pattern on both the front end and back end, then we could always get the right data with `state.currentClient.chain` and then use the proper config on the `req` on the back end

Reply:
Yeah, can do!

Reply:
Updated.